backend/app/services/diagnosis_agent.py

```
@@ -33,205 +33,529 @@ class DiagnosisState(TypedDict):
+                k_data = k_resp.json()
+                if k_data.get("documents"):
+                    doc = k_data["documents"][0]
+                    lat = float(doc["y"])
+                    lon = float(doc["x"])
+                    nx, ny = map_to_grid(lat, lon)
     except Exception as e:
-        return {"weather_data": {"temperature": "-", "humidity": "-", "wind_speed": "-", "precipitation_prob": "-"}}
+        print("Kakao API Error:", e)
+
+    # 2. 기상청 단기예보 조회 (약 3일치)
+    current_time = datetime.now()
```

Ruff (0.15.10)

[warning] `datetime.datetime.now()` called without a `tz` argument

(DTZ005)